frontend/src/views/Review.vue

```diff
@@ -11,24 +34,67 @@
 
 <script>
 import ReviewCard from '@/components/reviews-replies/ReviewCard'
+import ReviewForm from '@/components/reviews-replies/ReviewForm'
+import ReplyCard from '@/components/comments/CommentCard'
+import ReplyForm from '@/components/comments/CommentForm'
 import { mapGetters } from 'vuex'
 
 export default {
   components: {
-    ReviewCard
+    ReviewCard,
+    ReviewForm,
+    ReplyCard,
+    ReplyForm
   },
   props: {
-    id: String
+    courseID: String, // This is a query
+    reviewID: String // This is a param
+  },
+  data() {
+    return {
+      commentType: 'Reply'
```

Comment:
?

Reply:
This is used as the text for the button and comment prompt, i.e. it is currently either set to "Reply" or "Answer"